.gitignore

```diff
@@ -42,7 +43,8 @@ erl_crash.dump
 .LSOverride
 
 # Icon must end with two \r
-Icon


```

Comment:
This looks like a line ending issue, @doomspork any idea how to undo this change specifically? I didn't update this particular line in my editor.

Reply:
My best suggestion is going to be a git reset on this file. Ping me in Slack if you want to walk through it